src/rtk/features/overviewOrg/overviewOrgSlice.ts

```diff
@@ -181,10 +229,23 @@ const overviewOrgSlice = createSlice({
       }
     },
   },
+  extraReducers: (builder) => {
+    builder
+      .addCase(fetchOverviewOrgs.fulfilled, (state, action) => {
```

Comment:
You'll need to add own cases for fetchGivenOverviewOrgs and fetchReceivedOverviewOrgs.